packages/node-opcua-end2end-test/test/end_to_end/test_e2e_1170.ts

```diff
@@ -0,0 +1,79 @@
+const async = require("async");
+const should = require("should");
+const opcua = require("node-opcua");
+const OPCUAClient = opcua.OPCUAClient;
+const UserTokenPolicy = opcua.UserTokenPolicy;
+const { perform_operation_on_client_session } = require("../../test_helpers/perform_operation_on_client_session");
+// require("node-opcua-service-session").UserNameIdentityToken;
+
+const describe = require("node-opcua-leak-detector").describeWithLeakDetector;
+describe("Testing bug #1170", function() {
+
+    const port = 1170;
+    let server = null;
+    let endpointUrl = "";
+
+
+    before(async () => {
+
+
+        server = new opcua.OPCUAServer({
+            port
+        });
+
+        await server.start();
+
+        let endpoints = server._get_endpoints();
+        endpointUrl = endpoints[0].endpointUrl;
+
+
+    });
+    after(function(done) {
+        server.shutdown(done);
+    });
+
+    it("should productURI have the same value", (done) => {
```

Comment:
Please favor new style with async/await instead of callback ... code will be easier to read.

Reply:
Thanks, I just changed